CONTRIBUTING.md

```diff
@@ -149,6 +149,26 @@ The project contains end-to-end test which are meant to run against a real Kuber
 
 The end-to-end tests will create a set of dummy pods in the `default` namespace of the Lima cluster that will be used to inject different kind of failures and make different assertions against those injections.
 
+### Uploading test results to datadog
+
+In case you have a Datadog account and want to push the tests results to it, you can do the following:
```

Comment:
Do I want to push the test results?
----
When this runs in Gitlab, what key does it use to push results to DD? A team account API key?

Reply:
Per documentation [here](https://docs.datadoghq.com/continuous_integration/tests/junit_upload/?tab=linux#uploading-test-reports):

> for example, `local` when uploading results from a developer workstation, or ci when uploading them from a CI provider) in the DD_ENV environment variable. For example:

I followed such guidelines.

> 1. Do I want to push the test results?

Up to you TBH, if it helps detecting issue faster and giving timing comparison upon us, etc why you wouldn't want do it?

> 2. When this runs in Gitlab, what key does it use to push results to DD?

This does not run in GitLab but in CircleCI. The key used is a team key indeed.
That I'm also using locally TBH as I'm NOT allowed to create an API key (only an application key)

---

The keys does not really matter for the underlying instrumentation that is git based, it's only needed to have a key with enough permissions to upload an xml file.

Even if not ideal, I think it's OK for now

I still asked the question to have guidance around this.